app/views/legal/privacy-notice.html

```diff
@@ -3,87 +3,82 @@
 {% set hideButton = true %}
 
 {% block form %}
-<h2 class="govuk-heading-m">Who we are</h2>
+<p>When you have a DfE (Department for Education) Identity account, DfE collects and processes some of your personal data.<p>
 
-<p>Get an identity to access Teacher Services is run by the <a href="https://www.gov.uk/government/organisations/teaching-regulation-agency/about">Teaching Regulation Agency (TRA)</a>, an executive agency of the Department for Education (DfE).</p>
+<p>DfE is the data controller for your information, as defined by data protection law.</p>
 
-<p>For the purpose of General Data Protection Regulation (GDPR), DfE is the data controller for the data we hold and process.</p>
+<p>DfE is referred to as ‘we’, ‘us’ and ‘our’ in this privacy policy.</p>
 
-<h2 class="govuk-heading-m">What data we collect</h2>
+<h2 class="govuk-heading-m">What personal data we collect</h2>
 
-<p>We collect:</p>
+<p>When you create a DfE Identity account, we collect the following information about you:</p>
 <ul class="govuk-list govuk-list--bullet">
-  <li>names</li>
-  <li>email addresses</li>
-  <li>dates of birth</li>
+  <li>name</li>
+  <li>email address</li>
+  <li>mobile phone number</li>
+  <li>date of birth</li>
```

Comment:
I thought about this when doing it - the order reflects the data we collect if a user does not have a record on the DQT. So it's effectively a ‘worst case scenario’ of the maximum amount of data we would collect rather than doing a bespoke list depending on whether the integrating service is hooked up to the DQT.